src/datepicker/datepicker-config.ts

```diff
@@ -10,12 +10,14 @@ import {NgbDateStruct} from './ngb-date-struct';
 @Injectable()
 export class NgbDatepickerConfig {
   dayTemplate: TemplateRef<DayTemplateContext>;
+  displayMonths = 1;
   firstDayOfWeek = 1;
   markDisabled: (date: NgbDateStruct, current: {year: number, month: number}) => boolean;
   minDate: NgbDateStruct;
   maxDate: NgbDateStruct;
   outsideDays: 'visible' | 'collapsed' | 'hidden' = 'visible';
   showNavigation = true;
+  showNavigationSelect = true;
```

Comment:
Oh, haven't seen your comment there in time. 
I think I prefer to have one field in case we want to add other navigation types later (meaning `quickNavigation` won't work)